cx.ring.RingDaemon.json

```diff
@@ -0,0 +1,393 @@
+                    "path": "pjproject_patches/add_dtls_transport.patch"
+                },
+                {
+                    "type": "patch",
+                    "path": "pjproject_patches/fix_ioqueue_ipv6_sendto.patch"
+                },
+                {
+                    "type": "patch",
+                    "path": "pjproject_patches/fix_turn_fallback.patch"
+                },
+                {
+                    "type": "patch",
+                    "path": "pjproject_patches/gnutls.patch"
+                },
+                {
+                    "type": "patch",
+                    "path": "pjproject_patches/fix_base64.patch"
+                },
+                {
+                    "type": "patch",
+                    "path": "pjproject_patches/ice_config.patch"
+                },
+                {
+                    "type": "patch",
+                    "path": "pjproject_patches/notestsapps.patch"
+                },
+                {
+                    "type": "patch",
+                    "path": "pjproject_patches/ipv6.patch"
+                },
+                {
+                    "type": "patch",
+                    "path": "pjproject_patches/multiple_listeners.patch"
+                },
+                {
+                    "type": "patch",
+                    "path": "pjproject_patches/pj_ice_sess.patch"
+                },
+                {
+                    "type": "patch",
+                    "path": "pjproject_patches/rfc6062.patch"
+                }
+            ]
+        },
+        {
+            "name": "yaml-cpp",
+            "buildsystem": "cmake",
+            "build-options": {
+                "cflags": "-fPIC",
+                "cxxflags": "-fPIC",
+                "ldflags": "-fPIC"
+            },
+            "config-opts": [
+                "-DBUILD_STATIC=ON",
+                "-DBUILD_SHARED=OFF",
+                "-DYAML_CPP_BUILD_TOOLS=OFF",
+                "-DBUILD_SHARED_LIBS=OFF"
+            ],
+            "sources":[
+                {
+                    "type": "git",
+                    "url": "https://github.com/jbeder/yaml-cpp.git",
+                    "commit": "beb44b872c07c74556314e730c6f20a00b32e8e5"
+                }
+            ]
+        },
+        {
+            "name": "json-cpp",
+            "buildsystem": "cmake",
+            "config-opts":[
+                "-DBUILD_STATIC_LIBS=ON",
+                "-DBUILD_SHARED_LIBS=OFF",
+                "-DJSONCPP_WITH_TESTS=OFF",
+                "-DCMAKE_INSTALL_LIBDIR=lib",
+                "-DCMAKE_CXX_FLAGS=-fPIC",
+                "-DCMAKE_C_FLAGS=-fPIC"
+            ],
+            "sources":[
+                {
+                    "type": "git",
+                    "url": "https://github.com/open-source-parsers/jsoncpp",
+                    "branch": "1.8.3"
+                }
+            ]
+        },
+        {
+            "name": "dbus-c++",
```

Comment:
Sorry, I should've been more clear here. This also needs `"rm-configure": true,` in order to work on aarch64. I've just built down to that part on a scaleway aarch4 system so I know that'll work ;-)

Reply:
Ok thanks will do it